### PR TITLE
Fix SafariExtension Width

### PR DESCRIPTION
**Pull request by evancrow**
Needed to set a min-width for iPad devices where the HTML isn't automatically set to fill the space.

### How was this tested?
- [x] I tested this manually

### Screenshots and screen recordings
Before:
Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2022-10-31 at 09 27 55

After:
Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2022-10-31 at 09 32 26

### CODE REVIEW

File: NeevaForSafari/NeevaForSafari/ViewController.swift [hunk 1]
```diff
@@ -36,7 +36,7 @@ class ViewController: UIViewController, WKNavigationDelegate, WKScriptMessageHan
         // Override point for customization.
     }
 
-    func userContentController(didReceive message: WKScriptMessage) {
+    func userContentController(_ userContentController: WKUserContentController, didReceive message: WKScriptMessage) {
```
Comment (jedireza) [inline NeevaForSafari/NeevaForSafari/ViewController.swift:39]: Is this change necessary? We have a different function signature, but still a noop.

Comment (evancrow) [inline NeevaForSafari/NeevaForSafari/ViewController.swift:39]: BTW, this change was caused by the protocol being updated with the new version of Swift. I didn't catch the build failure until I ran NeevaForSafari, so all should be good now.

Comment (evancrow) [reply NeevaForSafari/NeevaForSafari/ViewController.swift:39]: Just left a comment as you asked :) (see below)
